packages/react/.storybook/main.js

```diff
@@ -0,0 +1,21 @@
+const rootMain = require('../../../.storybook/main');
```

Comment:
looking at this config, it isn't required ? since nothing custom is actually being done here